readme.markdown

```diff
@@ -213,27 +227,184 @@ query parameter. For example:
 
 ### nimble develop
 
-The ``develop`` command allows you to link an existing copy of a package into
-your installation directory. This is so that when developing a package you
-don't need to keep reinstalling it for every single change.
-
-    $ cd ~/projects/jester
-    $ nimble develop
+The develop command is used for putting packages in a development mode. When
+executed with a list of packages it clones their repository and if it is
+executed in a package directory it adds cloned packages to the special
+`nimble.develop` file. This is a special file which is used for holding the
+paths to development mode dependencies of the current directory package. It has
+the following structure:
+
+```json
+{
+    "version": "0.1.0",
+    "includes": [],
+    "dependencies": []
+}
+```
 
-Any packages depending on ``jester`` will now use the code in
-``~/projects/jester``.
+* `version` - JSON schema version
+* `includes` - JSON array of paths to included files.
+* `dependencies` - JSON array of paths to Nimble packages directories.
 
-If you specify a package name to this command, Nimble will clone it into the
-current working directory.
+The format for included develop files is the same as the project's develop
+file, but their validation works slightly different.
 
-    $ nimble develop jester
+Validation rules:
 
-The ``jester`` package will be cloned into ``./jester`` and it will be linked
-to your installation directory.
+* The included develop files must be valid.
+* The packages listed in `dependencies` section must be dependencies required
+by the package's `.nimble` file and to be in the required by its version range.
+Transitive dependencies are not allowed but this may be changed in the future.
+* The packages listed in the included develop files are required to be valid
+**Nimble** packages, but they are not required to be valid dependencies of the
+current project. In the last case, they are simply ignored.
+* The develop files of the develop mode dependencies a package are being
```

Comment:
```suggestion
* The develop files or the develop mode dependencies of a package are being
```

Not sure if I understood this one correctly; my suggestion might be wrong.

Reply:
Here is not `or`. `Of` is correct. Additional `of` is added.